Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/PDFDiscretePresentationController.mm

```diff
@@ -1306,9 +1306,12 @@ static float elasticDeltaForTimeDelta(float initialPosition, float initialVeloci
         asyncRenderer->setShowDebugBorders(showDebugBorders);
 }
 
-void PDFDiscretePresentationController::updateForCurrentScrollability(OptionSet<TiledBackingScrollability>)
+void PDFDiscretePresentationController::updateForCurrentScrollability(OptionSet<TiledBackingScrollability> scrollability)
 {
-
+    for (auto& row : m_rows) {
+        if (auto* tiledBacking = row.contentsLayer->tiledBacking())
+            tiledBacking->setScrollability(scrollability);
+    }
```

Comment:
@smfr does it make sense to update this for all rows? Or only for the visible row?

Reply:
Only the visible row, but when another row becomes visible you'll need to update its tiledBacking.